decidim-core/lib/tasks/decidim_tasks.rake

```diff
@@ -39,10 +39,15 @@ namespace :decidim do
   end
 
   desc "Applies upgrade modifications to the already installed application."
-  task upgrade_app: [:"decidim:remove_default_favicon"]
+  task upgrade_app: [:"decidim:remove_default_favicon", :"decidim:upgrade_ruby_version"]
 
   desc "Removes the default favicon from the application."
   task :remove_default_favicon do
     FileUtils.rm("public/favicon.ico", force: true)
   end
+
+  task :upgrade_ruby_version do
+    template_dir = "#{Gem.loaded_specs["decidim-generators"].full_gem_path}/lib/decidim/generators/app_templates"
+    FileUtils.cp("#{template_dir}/.ruby-version", ".")
+  end
```

Comment:
I don't think this is actually necessary, as if I try to `bundle update decidim` on a development_app generated with the release/0.27-stable branch with ruby 3.0.2, I got the following error message:
```
Bundler found conflicting requirements for the Ruby version:  
  In Gemfile:                                                 
    decidim was resolved to 0.28.0.dev, which depends on      
      Ruby (>= 3.1)                                           
                                                              
  Current Ruby version:                                       
    Ruby (= 3.0.2)           
```                              

So, at least in my case, I had to change the .ruby-version for the update to work. I'm using [asdf](https://asdf-vm.com/), so I don't know the case with others ruby version managers (ie rbenv)

Reply:
Can you check out if this is also the behavior on rbenv? 

These are the steps I was following to get this error:

```bash
git checkout release/0.27-stable
bundle                          
bin/rake development_app        
git checkout develop            
cd development_app              
bundle update decidim           
```

Reply:
same behavior.